python/GafferUI/SpreadsheetUI/_LinkedScrollBar.py

```diff
@@ -91,24 +91,47 @@ def __init__( self, orientation, scrolledContainers, **kw ) :
 			scrollBar.rangeChanged.connect( Gaffer.WeakMethod( self.__rangeChanged ) )
 			scrollBar.stepsChanged.connect( Gaffer.WeakMethod( self.__stepsChanged ) )
 
+		self.isUpdating = False
```

Comment:
This should be made private with an `__` prefix.

Reply:
Fixed and squashed into https://github.com/GafferHQ/gaffer/pull/5507/commits/d4b6571097e815b598b78142c73d1ea2d5d2f3e9